### PR TITLE
Add ble events

### PR DESCRIPTION
**Pull request by iliapenev**
Add ability to broadcast events on bluetooth state change.

### CODE REVIEW

File: src/esp32/esp32_bt.c [hunk 1]
```diff
@@ -386,6 +390,30 @@ struct mg_str esp32_bt_mbuf_to_flat(const struct os_mbuf *om) {
...
+    }
+    case ESP32_BT_STARTED: {
+      return MGOS_BT_STARTED;
+      break;
+    }
+    case ESP32_BT_STOPPING: {
+      return MGOS_BT_STOPPING;
+      break;
+    }
+    default: {
```
Comment (rojer) [inline src/esp32/esp32_bt.c]: let's drop the default case so if another state is added the compiler reminds us to handle it.

Comment (rojer) [reply src/esp32/esp32_bt.c]: also, breaks after return are not necessary

Comment (iliapenev) [reply src/esp32/esp32_bt.c]: default removed, sorry about break; copy/paste issue.

**Comment by rojer** [comment]
merged, thanks!

**Pull request merged**